### PR TITLE
Document more DCGET features and add a note to the DCID search page.

### PR DESCRIPTION
**Pull request by mvashishtha**

### CODE REVIEW

File: api/sheets/get_variable.md [hunk 1]
```diff
@@ -13,28 +13,63 @@ grand_parent: API
 Given a list of [Place](https://browser.datacommons.org/kg?dcid=Place) DCIDs, a [StatisticalVariable](https://browser.datacommons.org/kg?dcid=StatisticalVariable), and a date, get the measurements of that variable at those places on that date.
 
 **Arguments**
-*   `dcids` - place DCIDs
+*   `dcids` - place DCIDs as a single value, a row, or a column
 *   `variable` - The StatisticalVariable to get
-*   `date` - The date to observe, e.g. 2017, "2017", "2017-12"
+*   `date` - The dates to observe, e.g. 2017, "2017", "2017-12". This can be a single value, a row, or a column.
```
Comment (tjann) [inline api/sheets/get_variable.md:18]: Can maybe explain this is ISO 8601 format. For a split second I was thinking 2017-2012.

File: api/sheets/get_variable.md [hunk 2]
```diff
@@ -13,28 +13,63 @@ grand_parent: API
...
+
+`date` is optional. If it's not specified, the function returns the latest observation of the variable.
 
 **Returns**
 
-The value of the variable at those places on that date.
+The value of the variable at those places on that date (or on the latest available date, if no date is specified).
```
Comment (tjann) [inline api/sheets/get_variable.md:24]: nit: s/that/the specified/

File: api/sheets/get_variable.md (hunk 1 above)
Comment (tjann) [reply api/sheets/get_variable.md:18]: Nit: should dcid and date arguments have the same plurality?

Comment (mvashishtha) [reply api/sheets/get_variable.md:18]: I specified that it is ISO 8601 format. I should change "date" to "dates", but that requires changing the app. I filed b/157822238 for that.

File: api/sheets/get_variable.md (hunk 2 above)
Comment (mvashishtha) [reply api/sheets/get_variable.md:24]: Done.

File: api/sheets/get_variable.md (hunk 1 above)
Comment (mvashishtha) [reply api/sheets/get_variable.md]: Also, the function can take a Google Sheets date object instead of a string, so I specified that.

Comment (mvashishtha) [reply api/sheets/get_variable.md]: Also, the function can take a Google Sheets date object instead of a string, so I specified that.

**Pull request merged**